<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPost</class>
 <widget class="QDialog" name="DialogPost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color:white;
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color:#f0efef;
	border:2px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit

{
font: 700 11pt &quot;Segoe UI&quot;;
	background-color:white;
	color:#010000;
	border:0;
}</string>
   </property>
   <property name="text">
    <string>Other brands</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vwbutton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:white;
border: 2px solid black;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/Screenshot 2025-02-05 143551.png</normaloff>../../Pictures/Screenshot 2025-02-05 143551.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="audibutton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:white;
border: 2px solid black;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/Screenshot 2025-02-05 144541.png</normaloff>../../Pictures/Screenshot 2025-02-05 144541.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="opelbutton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:white;
border: 2px solid black;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/Screenshot 2025-02-05 144648.png</normaloff>../../Pictures/Screenshot 2025-02-05 144648.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="fordbutton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:white;
border: 2px solid black;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/Screenshot 2025-02-05 144852.png</normaloff>../../Pictures/Screenshot 2025-02-05 144852.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bmwbutton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:white;
border: 2px solid black;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/Screenshot 2025-02-05 165013.png</normaloff>../../Pictures/Screenshot 2025-02-05 165013.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="mercedesbutton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:white;
border: 2px solid black;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/Screenshot 2025-02-05 165059.png</normaloff>../../Pictures/Screenshot 2025-02-05 165059.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit

{
font: 700 11pt &quot;Segoe UI&quot;;
	background-color:white;
	color:#010000;
	border:0;
}</string>
   </property>
   <property name="text">
    <string>Popular brands</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit

{
font: 700 11pt &quot;Segoe UI&quot;;
	background-color:white;
	color:#010000;
	border:0;
}</string>
   </property>
   <property name="text">
    <string>Model</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit

{
font: 700 11pt &quot;Segoe UI&quot;;
	background-color:white;
	color:#010000;
	border:0;
}</string>
   </property>
   <property name="text">
    <string>Fuel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>500</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit

{
font: 700 11pt &quot;Segoe UI&quot;;
	background-color:white;
	color:#010000;
	border:0;
}</string>
   </property>
   <property name="text">
    <string>Power</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit

{
font: 700 11pt &quot;Segoe UI&quot;;
	background-color:white;
	color:#010000;
	border:0;
}</string>
   </property>
   <property name="text">
    <string>Transimission</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>500</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit

{
font: 700 11pt &quot;Segoe UI&quot;;
	background-color:white;
	color:#010000;
	border:0;
}</string>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>540</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Post</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color:#f0efef;
	border:2px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>440</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color:#f0efef;
	border:2px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_11">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color:#f0efef;
	border:2px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_12">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>550</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color:#f0efef;
	border:2px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_13">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>550</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color:#f0efef;
	border:2px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>To menu</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_14">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
font: 700 11pt &quot;Segoe UI&quot;;
	background-color:#f0efef;
	border:2px;
}</string>
   </property>
   <property name="text">
    <string>    Posted succesfully!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
